doc/thing/samples/basic-samples/thermo.rst

```diff
@@ -0,0 +1,54 @@
+thermo
+======
+
+Overview
+--------
+
+The thermo example simulates a device showing the room temperature.
+The purpose of this sample is to show how to register an integer value
+as a sensor using the KNoT Zephyr SDK.
+
```

Comment:
Please, make it clear that the value is automatically updated. And how it is updated

Reply:
Ok.